app/src/main/java/net/kuama/documentscanner/domain/FindPaperSheetContours.kt

```diff
@@ -68,7 +69,16 @@ class FindPaperSheetContours : UseCase<PaperSheetContoursResult, FindPaperSheetC
             }
 
             val result: PaperSheetContoursResult = contours.firstOrNull()?.let {
-                PaperSheetContoursResult(params.bitmap, Corners(it.shape.toList(), original.size()))
+                var corners = Corners(
```

Comment:
Maybe better to build corners already with sorted points?
`val corners = CornerFactory.createSorted(shape, size)`